### ISSUE TITLE
Waymo pointpillar model constant loss and wrong prediction issue

### ISSUE THREAD

**Issue by menayl123** [waymo-research/waymo-open-dataset#511]
<img width="312" alt="loss curve" src="https://user-images.githubusercontent.com/104393327/173119423-7a1bd194-31c5-4ddb-b8d1-6eb309a2ab4e.PNG">

Hi,
I am training pointpillar model using waymo dataset. I want to detect 1 class vehicle. I am using ssd as detection head. My loss intially decreases but later becomes constant and my prediction is wrong. I have attached my loss curve. Any idea what should I look into to find the issue? I have increased learning rate as well and changed the optimizer from SGD to ADAM but still face the same issue. Any hint would be a great help. Thanks
